strapi/strapi
# Return newly created IDs when using createMany

### What does it do?

Fixes #13856 by adding the array of IDs belonging to newly created entries when using `createMany`.

### Why is it needed?

At the moment, the `createMany` function only returns the count of created entries, making it quite hard for devs to perform actions on those elements in lifecycles or elsewhere in the code.

### How to test it?

Run a `createMany` query and notice how the returned object contains `count` and the new member `ids`.

### Related issue(s)/PR(s)

Depends on #14267 or it won't work on SQLite.
